recce/apis/check_api.py

```diff
@@ -38,15 +39,50 @@ def create_check_from_run(name, description, run_id):
     return None
 
 
+def create_check_from_schema(name, description, model_id):
+    def get_manifests_by_id(unique_id):
+        from recce.server import dbt_context
+        curr_manifest = dbt_context.get_manifest(base=False)
+        base_manifest = dbt_context.get_manifest(base=True)
+        if unique_id in curr_manifest.nodes.keys() or unique_id in base_manifest.nodes.keys():
+            return {
+                'current': curr_manifest.nodes.get(unique_id),
+                'base': base_manifest.nodes.get(unique_id)
+            }
+        return None
+
+    manifests = get_manifests_by_id(model_id)
+    if manifests is None:
+        return None
+
+    node = manifests['current'] or manifests['base']
+    if name is None:
+        name = f"Schema for model {node.name} {datetime.utcnow().isoformat()}"
+    params = {
+        "model_id": model_id,
+        "model_name": node.name,
+    }
+
+    check = Check(name=name, description=description, type=RunType.SCHEMA_DIFF, params=params)
+    return check
+
+
 @check_router.post("/checks", status_code=201, response_model=CreateCheckOut)
 async def create_check(check: CreateCheckIn):
-    if check.run_id is None:
-        raise HTTPException(501, "Not Implemented")
-    else:
+    if check.run_id:
+        # Create check from run
         check_record = create_check_from_run(check.name, check.description, check.run_id)
         if not check_record:
             raise HTTPException(status_code=404, detail=f"Run ID '{check.run_id}' not found")
         checks_db.append(check_record)
+    elif check.model_id:
```

Comment:
Model id is schema-check specific param, please put in `check.params.check_id`

Reply:
change to use `node_id`